src/scene/shader-lib/chunks/lit/frag/shadowPCSS.js

```diff
@@ -0,0 +1,185 @@
+export default /* glsl */`
```

Comment:
please add references to what this is based on as comments

Reply:
Done.